android/vcpkg_config_client/vcpkg.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "boinc-client",
+    "version-string": "7.19.0",
+    "builtin-baseline":"bd1ef2df46303989eeb048eb7aa9b816aa46365e",
+    "dependencies":
+    [
+        {
+          "name": "curl",
```

Comment:
When we are building current master with curl, we build it with `openssl` feature only: `curl[core,openssl]:armv6-android -> 7.82.0`
After this change additional configuration is passed to curl that will have negative impact: `curl[core,non-http,openssl,ssl]:armv6-android -> 7.82.0`
This should be fixed here and in two other vcpkg.json files

Reply:
Cannot do it with manifest mode.

Reply:
You can do it in manifest mode. E.g. https://github.com/BOINC/boinc-apps/blob/master/boinc-autodock-vina/vcpkg.json#L12-L15